### PR TITLE
LPS-192162  | Test Fix | Master

### PR DESCRIPTION
**Pull request by sirfabios**
https://liferay.atlassian.net/browse/LPS-192162

### CODE REVIEW

**Comment by sirfabios** [comment]
ci:test:sf

**Comment by timpak** [comment]
@sirfabios there isn't a generic path for this? I'm almost certain there should be since this feature is highly likely to be used by other component teams, like Echo.

**Comment by sirfabios** [comment]
Hi @timpak 
I hadn't found com, so I made this path.
I will search again after I finish the activities I'm doing.

**Comment by sirfabios** [comment]
Hi @timpak 
I googled, found this NavBar#NAV_ITEM_REDIRECTION, he sees that the object name is there, but doesn't click.

**Comment by sirfabios** [comment]
I resent a generic path.

**Comment by sirfabios** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/paths/VisualizeObjectDisplayPage.path [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	</tr>
 	<tr>
 		<td>SELECT_OBJECT</td>
-		<td>//span[contains(text(),'${key_pluralLabelName}')]</td>
+		<td>//div[contains(@class,'collapse menubar-collapse')]//a[contains(.,'${key_pluralLabelName}')]</td>
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/paths/VisualizeObjectDisplayPage.path:20]: Do we need to include `collapse` the path? Can we just use:
`//div[contains(@class,'menubar-collapse')]//a[contains(.,'${key_pluralLabelName}')]`

**Review by timpak — CHANGES_REQUESTED** [review]
@sirfabios please see my suggestion below.

**Comment by sirfabios** [comment]
ci:test:sf

**Comment by timpak** [comment]
ci:forward

**Comment by timpak** [comment]
ci:forward

**Pull request closed without merging**